src/libraries/System.Drawing.Common/tests/mono/System.Drawing/GraphicsTests.cs

```diff
@@ -2106,7 +2106,7 @@ public void MeasureCharacterRanges_NullStringFormat()
                     new CharacterRange (2, 1)
                 };
 
-        Region[] Measure(Graphics gfx, RectangleF rect)
+        Region[] Measure_Helper(Graphics gfx, RectangleF rect)
```

Comment:
Can this be moved to be a local function inside Measure below?

Reply:
Oh, nevermind, it's used by multiple tests.

Reply:
I didn't do it that way because the `MeasureLimits` test also uses it, but if you'd rather have the same local function in both test methods, I can move it.

Reply:
What you have is fine.  I hadn't noticed initially it was used by multiple tests.